src/BlazorShared/Authorization/Constants.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BlazorShared.Authorization
+{
+    public class Constants
```

Comment:
Might as well make this static too
```suggestion
    public static class Constants
```